src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR = 6;
+
+    public static String colorGenerator() {
```

Comment:
let's not create static methods - they are in general bad practice
also, use verbs for method names

```suggestion
    public String getRandomColor() {
```